install/kubernetes/cilium/values.yaml

```diff
@@ -165,12 +165,14 @@ global:
     # writing of the CNI configuration is performed by external automation.
     customConf: false
 
-    # confPath is the path to the CNI configuration directory on the host
-    confPath: /etc/cni/net.d
```

Comment:
This is used in https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/charts/agent/templates/daemonset.yaml#L348 , lack of this seem to be causing helm lint to fail.

Reply:
🤦 so sorry about that, this was not intentional! I omitted this change 👍